src/TSTransformer/classes/RoactSymbolManager.ts

```diff
@@ -25,7 +25,9 @@ export class RoactSymbolManager {
 
 		for (const symbolName of Object.values(ROACT_SYMBOL_NAMES)) {
 			const symbol = roactExports.get(ts.escapeLeadingUnderscores(symbolName));
-			assert(symbol);
+			if (!symbol) {
+				assert(false, `RoactSymbolManager could not find Roact export symbol for ${symbolName}`);
+			}
```

Comment:
Why not `assert(symbol, "...")`?

Reply:
TS for some reason does not typecheck that properly. It had to be an if statement.

Reply:
To be exact, the following error gets thrown:
```
'symbol' implicitly has type 'any' because it does not have a type annotation and is referenced directly or indirectly in its own initializer.ts(7022)
```